engines/stark/formats/iss.cpp

```diff
@@ -106,6 +109,31 @@ Audio::RewindableAudioStream *makeISSStream(Common::SeekableReadStream *stream,
 		size = (uint32)strtol(codec.c_str(), 0, 10);
 
 		return new ISSADPCMStream(stream, DisposeAfterUse::YES, size, freq, channels, blockSize);
+	} else if (codec.equals("Sound")) {
+
+		readString(stream);
+		// name ?
+
+		codec = readString(stream);
+		// sample count ?
+
+		codec = readString(stream);
+		channels = (uint16)strtol(codec.c_str(), 0, 10) + 1;
+
+		readString(stream);
```

Comment:
This value defines if the samples are 8 or 16 bits (0 for 8, 1 for 16). I don't know if there are actually 8 bits sounds in the game.